src/TypesFinder/FindTypeFromAst.php

```diff
@@ -17,13 +18,8 @@ class FindTypeFromAst
      * @param string $namespace
      * @return \phpDocumentor\Reflection\Type|null
      */
-    public function __invoke($astType, LocatedSource $locatedSource, $namespace = '')
+    public function __invoke(Context $context, $astType)
```

Comment:
I know `FindTypeFromAst` is used outside the context of just reflection, so strong :+1: on this; documentation on how to change existing code for the BC must be provided.